sql_generators/funnels/configs/login_funnels.toml

```
@@ -0,0 +1,72 @@
+destination_dataset = "firefox_accounts"
```

Let's write this to the `_derived` dataset

```suggestion
destination_dataset = "firefox_accounts_derived"
```